runtime/src/bank/tests.rs

```diff
@@ -12633,7 +12633,7 @@ fn test_distribute_partitioned_epoch_rewards() {
         .map(|_| StakeReward::new_random())
         .collect::<Vec<_>>();
 
-    let stake_rewards = hash_rewards_into_partitions(stake_rewards, &Hash::new(&[1; 32]), 100);
+    let stake_rewards = hash_rewards_into_partitions(stake_rewards, &Hash::new(&[1; 32]), 2);
```

Comment:
when we moved to using partitions by hash, the argument of `100` was incorrectly put on this test fn. When I fix the math in `distribute_partitioned_epoch_rewards`, the test fails.

Reply:
Nice catch.